sdk/core/core-tracing/src/interfaces.ts

```diff
@@ -215,6 +215,30 @@ export interface ContextAPI {
   active(): Context;
 }
 
+/**
+ * Returns true of the given {@link SpanContext} is valid.
+ * A valid {@link SpanContext} is one which has a valid trace ID and span ID as per the spec.
+ *
+ * @param context - the {@link SpanContext} to validate.
+ *
+ * @returns true if the {@link SpanContext} is valid, false otherwise.
+ */
+export function isSpanContextValid(context: SpanContext): boolean {
```

Comment:
Is it worth re-exporting these? Shouldn't we just have the SDKs ask OT directly?

Reply:
Agree that we are at a point where these thin wrappers around OT are less and less useful. They did save me a bunch of work when things moved to the `trace` namespace, but that should not happen anymore now that OT is 1.0...

Three reasons why I did not just rip that out now:

1. This is already a massive PR, so I wanted to keep changes manageable
2. We did a lot of work to remove @opentelemetry/api as a dependency from individual packages. I _think_ that what we would need to do here is not big - only core-http and core-rest-pipeline should really need any of these methods. But I wasn't ready to add @opentelemetry/api back as dependency for those yet
3. Biggest reason I held off was  https://github.com/Azure/azure-sdk-for-js/pull/14618 - I need to think about it a bit

Reply:
Created #16359 to track this